purchase_work_acceptance/models/account_invoice.py

```diff
@@ -0,0 +1,169 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+from odoo.tools.float_utils import float_compare
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    wa_id = fields.Many2one(
+        comodel_name='work.acceptance',
+        string='WA Reference',
+        domain=[('state', '=', 'accept')],
+        store=True,
```

Comment:
Why store=True? Is this computed?

Reply:
Add some help as following (you can use this help to all wa_id field, it will help user knows what it is used for).
help:
To control quantity and unit price of the vendor bill, to be according to the quantity and unit price of the work acceptance.